src/main/java/co/lotc/lever/cmd/Countdown.java

```diff
@@ -0,0 +1,52 @@
+package co.lotc.lever.cmd;
+
+import static org.bukkit.ChatColor.*;
+
+import co.lotc.core.bukkit.util.Run;
+import co.lotc.core.command.annotate.Arg;
+import co.lotc.core.command.annotate.Default;
+import co.lotc.core.command.annotate.Flag;
+import co.lotc.core.command.annotate.Range;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+import co.lotc.core.bukkit.util.LocationUtil;
+import co.lotc.lever.BaseCommand;
+
+import java.util.*;
+import java.util.List;
+
+public class Countdown extends BaseCommand {
+    public static final String PEX_COUNTDOWN_RADIUS = "lever.countdown.radius";
```

Comment:
Permission not pex

Reply:
thats my fault i use PEX as shorthand everywhere in lEver